### PR TITLE
Add a deployment guide for Render

### PR DESCRIPTION
**Pull request by emekaonu**
This PR introduces a guide for deploying Vue.js apps on the `Render` platform. [Render](https://render.com) completely frees up developers and teams from managing infrastructure by abstracting over complexities and minimizing cognitive overhead for developers.

### CODE REVIEW

**Comment by anurag** [comment]
In case it helps (and as additional context) [Gatsby](https://www.gatsbyjs.org/docs/deploying-to-render/) and [Create React App](https://github.com/facebook/create-react-app/pull/6695) have both added Render as an official deployment options.

File: docs/guide/deployment.md [hunk 1]
```diff
@@ -408,3 +408,24 @@ Deploy your application using nginx inside of a docker container.
...
+
+Use the following values during creation:
+
+ - **Environment:** `Static Site`
+ - **Build Command:** `npm run build` or `yarn build`
+ - **Publish directory:** `dist`
+
+That’s it! Your app will be live on your Render URL as soon as the build finishes.
+
+In order to receive direct hits using `history mode` on Vue Router, you need to add the following redirect rules in the `Redirects` tab under your site.
```
Comment (NataliaTepluhina) [inline docs/guide/deployment.md]:
```suggestion
In order to receive direct hits using `history mode` on Vue Router, you need to add the following redirect rules in the `Redirects/Rewrites` tab under your site.
```
It seems Render changed its UI a bit 😅

File: docs/guide/deployment.md [hunk 2]
```diff
@@ -408,3 +408,24 @@ Deploy your application using nginx inside of a docker container.
...
+
+Select `Redirects`, click on `Add Redirect Rule` button and
```
Comment (NataliaTepluhina) [inline docs/guide/deployment.md]:
```suggestion
Select `Redirects/Rewrites`, click on `Add Rule` button and
```

File: docs/guide/deployment.md [hunk 3]
```diff
@@ -408,3 +408,24 @@ Deploy your application using nginx inside of a docker container.
...
+
+ - **Source Path:** `/*`
+ - **Destination Path:** `/index.html`
+ - **Status:** `200`
```
Comment (NataliaTepluhina) [inline docs/guide/deployment.md]: Right now `Status` has two options only: `Redirect` or `Rewrite`

Comment (anurag) [reply docs/guide/deployment.md]: Thanks @NataliaTepluhina! Will submit the suggested changes very soon.

**Comment by anurag** [comment]
@NataliaTepluhina I just submitted a new PR which addresses these issues. Feel free to close this one.

**Comment by NataliaTepluhina** [comment]
@anurag awesome! Closing this in favor of https://github.com/vuejs/vue-cli/pull/4217

**Pull request closed without merging**